sebs/aws/s3.py

```
@@ -46,11 +98,37 @@ def __init__(
+        """Correct bucket name for S3 naming requirements.
+        
+        Args:
+            name: Original bucket name
+            
+        Returns:
+            str: Corrected bucket name (no changes for S3)
+        """
         return name
 
     def _create_bucket(
         self, name: str, buckets: List[str] = [], randomize_name: bool = False
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)